### PR TITLE
bears/gettext: Add DennisBear

### PR DESCRIPTION
**Pull request by Flap-Py**
Linter bear for translated PO and POT files.

closes https://github.com/coala/coala-bears/issues/892

### CODE REVIEW

File: bears/gettext/DennisBear.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    """
+    Lints your translated PO and POT files!
+
+    Check multiple lint rules on all the strings in the PO file
+    generating a list of errors and a list of warnings.
+
+    See http://dennis.readthedocs.io/en/latest/linting.html for
+    list of all error codes.
+
+    http://dennis.readthedocs.io/
```
Comment (Nosferatul) [inline bears/gettext/DennisBear.py]: For more details, check : `....`.

Comment (Nosferatul) [inline bears/gettext/DennisBear.py]: Also, write tests

**Comment by Nosferatul** [comment]
Why do you have `__init__.py` file in the commit?

**Comment by Flap-Py** [comment]
@Nosferatul its there in every folder in `bears` and I was told to do so https://github.com/coala/coala-bears/issues/892
Well I missed the unit tests though :disappointed:

**Comment by Nosferatul** [comment]
It was ok, you just needed tests, why did you closed it?

**Comment by Flap-Py** [comment]
the ci tests were failing, I thought unit tests were the reason so I closed.
No worries I would pr with unit tests :)

**Comment by jayvdb** [comment]
You can push `--force` to this branch when you have tests ready.

**Comment by Flap-Py** [comment]
Closing this pr, and making new one.

**Pull request closed without merging**